### PR TITLE
Ignore 115 errors

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
`115 operation in progress` isn't a fatal error condition, so let's not treat it as such.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -487,7 +487,9 @@ private function sendRawRequest(string $host, int $port, string $rawRequest)
             socket_set_option($this->socket, SOL_SOCKET, SO_RCVTIMEO, ['sec' => $this->readTimeout, 'usec' => 0]);
             @socket_connect($this->socket, $host, $port);
             $socketErrorCode = socket_last_error($this->socket);
-            if ($socketErrorCode) {
+            if ($socketErrorCode === 115) {
+                $this->logger->info('Bedrock\Client - socket_connect returned error 115, continuing.');
```
Comment (iwiznia) [inline src/Client.php]: I don't get this, just continue as if it succeeded? Don't you need to throw a `ConnectionFailure` so we retry in another server?

Comment (righdforsa) [reply src/Client.php]: Correct, we're just going to continue as if it succeeded.

**Comment by flodnv** [comment]
FYI, this results in a lot of PHP Warnings: https://github.com/Expensify/Expensify/issues/96847

**Comment by iwiznia** [comment]
> FYI, this results in a lot of PHP Warnings: Expensify/Expensify#96847

Of course it did, this change makes no sense. I should've rejected instead of commented...

I see 0 tests here and it was CPed to prod...

**Comment by quinthar** [comment]
As discussed in https://github.com/Expensify/Expensify/issues/96847#issuecomment-455242389 this is known and intentional.  I think we can close this.

**Comment by iwiznia** [comment]
We can solve the warning, plus if you close that issue, botify will reopen it.

**Pull request merged**